<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailWindow</class>
 <widget class="QWidget" name="DetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Detail</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="info_layout">
     <item>
      <widget class="QLabel" name="titel_label">
       <property name="text">
        <string>Task Title</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="status_layout">
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="status_menu">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Pending</string>
         </property>
         <item>
          <property name="text">
           <string>Pending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Finished</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In Progress</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="description_label">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="description_entry"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>200</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="back_button">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
